packages/@aws-cdk/aws-serverless/README.md

```diff
@@ -1,5 +0,0 @@
-## AWS Serverless Construct Library
```

Comment:
My diff only shows 1 file removed from this package. Is that right?

Reply:
The rest were identified as renamed. I changed the content of the README so that's why it shows up as removed, or so I assume.